specs/core/1_custody-game.md

```diff
@@ -115,9 +67,7 @@ This document details the beacon chain additions and changes in Phase 1 of Ether
 | - | - |
 | `MAX_CUSTODY_KEY_REVEALS` | `2**4` (= 16) |
```

Comment:
I think we need more than 16 `MAX_CUSTODY_KEY_REVEALS`. Every validator reveals once per 2048-epoch period, so in the worst case that's 2**22 validators revealing once every 2**16 slots, or 64 per slot. So the max should probably be 256.